src/main/java/bio/terra/testrunner/runner/config/ClusterSpecification.java

```diff
@@ -13,6 +13,7 @@
   public String project;
   public String namespace;
   public String containerName;
+  public ApplicationSpecification application;
```

Comment:
I don't think we should move the whole `ApplicationSpecification` object to the `ClusterSpecification`, only the properties that are tied to the server environment:
```
  public String componentLabel = "app.kubernetes.io/component";
  public String apiComponentLabel = "api";
```
The other properties of `ApplicationSpecification` seem like things we may want to change for each `TestConfiguration`. Those two properties above though, I think will only vary per server, not per test config. What do you think?

Reply:
Makes sense. I put the rest of the `ApplicationSpecification` back into test config.